README.md

```diff
@@ -222,35 +222,70 @@ Grade.deleteAll { error in
 }
 ```
 
-### Customization
+### Customizing your Model
 
-The ORM uses [Swift-Kuery](https://github.com/IBM-Swift/Swift-Kuery) which allows you to customize and execute your own queries without breaking any existing ORM functionality. You'll want to have access to the table for your object, which you can get with the `getTable()` function:
+The ORM defines an extension to `Model` which provides a number of `public static executeQuery(…)` functions. These methods can be used to create custom functions within your model’s that perform more complex database operations. The example below defines and Info type and adds a custom method to it that will retrieve all database records which have an age over 20:
```

Comment:
Typo: `model's` -> `model`
Typo: `These methods` -> `These functions`
Typo: `and Info type` -> `an Info type`
... I might rename the model to Person, and reword the final sentence as:
> The example below defines a Person model with a custom function to retrieve all records which have `age > 20`:

Reply:
Done.